devel is future 2.x

kdelee opened the pull request:
1.x is now living on a stable branch

There are some other places like release notes that we will have to update at release time, but for now this will at least make devel report as a different version than 1.4.6

kdelee committed:
devel is future 2.x

1.x is now living on a stable branch

kdelee commented:
recheck

jladdjr commented:
Posted PR to clear test flake:
https://github.com/ansible/ansible-runner/pull/472

kdelee committed:
drop py2 target

More remains to fully drop py2 but 2.0.0 does not intend to support it as a version of python

kdelee commented:
waiting for #472 to go in and then will rebase before gate

kdelee commented:
:confused: they all passed but then the check failed?

jladdjr commented:
yeah, I was just going to was.. `POST_FAILURE`?

jladdjr commented:
recheck
